workflows/phylogenetics/wf_find_shared_variants.wdl

```diff
@@ -0,0 +1,40 @@
+version 1.0
+
+import "../../tasks/utilities/task_file_handling.wdl" as file_handling
+import "../../tasks/phylogenetic_inference/task_shared_variants.wdl" as shared_variants_task
+import "../../tasks/task_versioning.wdl" as versioning
+
+workflow shared_variants_wf {
+  meta {
+    description: "Concatenate variants from multiple samples and create a shared variants table"
+  }
+  input {
+    Array[File] files_to_cat
```

Comment:
I'd rename this from files_to_cat to a more specific name. This makes it seem to me that I could through anything in there.

Reply:
Revised to variants_to_cat